.travis.yml

```diff
@@ -25,6 +25,7 @@ install: source continuous_integration/travis/install.sh
 script:
   - source continuous_integration/travis/run_tests.sh
   - flake8 dask
+  - pycodestyle --select=E722 --exclude=tests dask  # check for bare `except:` blocks
```

Comment:
Looks like you fixed them in `tests` directories as well. Can the `exclude` be removed?